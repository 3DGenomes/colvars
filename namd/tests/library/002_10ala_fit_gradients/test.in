colvarsTrajFrequency    1

indexFile ../Common/da.ndx


colvar {
    name eigenvector_Ca

    outputTotalForce on
    outputAppliedForce on
    width 1.0 

    eigenvector {
        #    debugGradients
        atoms {
            enableFitGradients yes
            indexGroup Protein_C-alpha
        }
        refpositionsfile ../Common/pi-ideal.xyz
        #    vector { (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) (3.0, 1.0, 2.0) (2.0, 3.0, 1.0) (1.0, 2.0, 3.0) }
        vectorFile ../Common/da.xyz
        differenceVector yes
    }
} 


harmonic {
    colvars        eigenvector_Ca
    forceConstant  10.0  
    centers        0.0
}
