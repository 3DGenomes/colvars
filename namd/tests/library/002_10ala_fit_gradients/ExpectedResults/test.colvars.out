colvars: ----------------------------------------------------------------------
colvars: Using NAMD interface, version 2014-10-26.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = eigenvector_Ca
colvars:   Initializing a new "eigenvector" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/pi-ideal.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating the vector projection: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     # vector =  [default]
colvars:     # vectorFile = ../Common/da.pdb
colvars:     # vectorCol = "" [default]
colvars:     Geometric center of the provided vector: ( 0.9999 , 2 , 3 )
colvars:     # differenceVector = on
colvars:     "differenceVector" is on: subtracting the reference positions from the provided vector: v = v - x0.
colvars:     "differenceVector" is on: normalizing the vector.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { eigenvector_Ca }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 10
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
